### ISSUE TITLE
'Automatically make log entry' not working

### ISSUE THREAD

**Issue by texaspete** [Marginal/EDMarketConnector#119]
What can I try when 'Automatically make a log entry on entering a system' is not working? That is to say, EDMC does not detect when I jump to a new system. I have to remember to trigger it manually.

I am running Elite Dangerous Horizons (64-bit). My elite-dangerous-64 folder can is located at D:\Program Files (x86)\Frontier\EDLaunch\Products and contains both an AppConfig.xml file and a Logs subfolder.

Extra clue: Whilst I do have Steam installed, Elite Dangerous was not purchased via Steam and does not exist in my steamapps folder.

**Comment by Athanasius** [comment]
Presumably this is because the format of the log entries changed, both to accommodate listing the system co-ordinates, and also some other minor changes.

EDMC will need an update to handle this, and then send the listed co-ords to EDSM.

**Comment by texaspete** [comment]
So everyone is affected?

**Comment by AnthorNet** [comment]
Yes, the pull request have been made to handle the new logs already.
See #112
